tests/virtiofs/datavolume.go

```diff
@@ -335,5 +337,116 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 			Expect(err).ToNot(HaveOccurred())
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 		})
+
+		It("[Serial] should be successfully started and injected into the container", Serial, func() {
```

Comment:
Why is it serial again?

Reply:
I'm not sure what you mean, IIUC since we need to enable a FG this must be marked as serial, right?

Reply:
We could enable the feature gate in the tests by default, so this can be parallelized, see tests/testsuite/kubevirtresource.go